conf/messages.cy

```diff
@@ -581,3 +581,15 @@ appealCoverBothPenalties.p1 = Mae hyn yn eich galluogi i nodi manylion apêl unw
 incompleteSessionData.headingAndTitle = Mae’n ddrwg gennym, mae rhywbeth wedi mynd o’i le
 incompleteSessionData.p1 = Mae problem dechnegol wedi codi. Bydd yn rhaid i chi ddechrau’ch apêl eto. Nid ydym wedi cadw’ch data.
 incompleteSessionData.link = Dychwelyd i’r dudalen gychwyn ar gyfer apeliadau
+
+# Appeal by letter kick out page
+# -----------------------------------------------------------
+appealByLetter.headingAndTitle = You can appeal this penalty by letter
+appealByLetter.p1 = To ask HMRC to review this penalty, send us:
+appealByLetter.li.1.1 = details of the steps taken to cancel the VAT registration
+appealByLetter.li.1.2 = dates of any contact with HMRC on this matter
+appealByLetter.li.1.3 = any other information you’d like to include to support an appeal
+appealByLetter.p2 = If HMRC accepts your appeal, we will remove:
+appealByLetter.li.2.1 = this penalty
+appealByLetter.li.2.2 = all other VAT late submission or late payment penalties for the same period
+appealByLetter.inset.1 = Ysgrifennwch at:
```

Comment:
I think we should leave it completely 'untranslated', saves having wrong translations